### PR TITLE
[#175059122] Centralise config read

### PR DESCRIPTION
**Pull request by balanza**
This PR introduces the `utils/config` module which aims to be the unique interface for application configuration. 

Every place in which the application used to directly access env variable has been refactored to use this module instead.

### CODE REVIEW

**Comment by gunzip** [comment]
lgtm, if you handle the FIXMEs we can proceed and merge it

**Review by AleDore — COMMENTED** [review]
lgtm. Maybe we have to check if there are any case of "optional" env variable, and how we can handle it using this approach.

**Comment by gunzip** [comment]
lgtm, why draft ?

File: DeleteUserDataActivity/index.ts [hunk 1]
```diff
@@ -10,21 +8,23 @@ import { MESSAGE_STATUS_COLLECTION_NAME } from "io-functions-commons/dist/src/mo
...
 import { NotificationDeletableModel } from "../utils/extensions/models/notification";
 import { NotificationStatusDeletableModel } from "../utils/extensions/models/notification_status";
 import { ProfileDeletableModel } from "../utils/extensions/models/profile";
 
-const cosmosDbName = getRequiredStringEnv("COSMOSDB_NAME");
+const config = getConfigOrThrow();
+const cosmosDbName = config.COSMOSDB_NAME;
 
 const messagesContainer = cosmosdbClient
   .database(cosmosDbName)
```
Comment (BurnedMarshal) [inline DeleteUserDataActivity/index.ts:22]: Can we use directly the config value?
```suggestion
const messagesContainer = cosmosdbClient
  .database(config.COSMOSDB_NAME)
```

**Review by BurnedMarshal — COMMENTED** [review]
lgtm, only check if is better variables aliasing instead using directly the values

**Pull request merged**